test/integration/crcsuite/crcsuite.go

```diff
@@ -247,6 +249,48 @@ func CheckOutputMatchWithRetry(retryCount int, retryTime string, command string,
 	return matchErr
 }
 
+// CheckCRCStatus checks that output of status command
+// matches given regex number of consecutive times
+func CheckCRCStatus(state string) error {
+
```

Comment:
unwanted new line

Reply:
I'm not sure if it's unwanted (gofmt doesn't mind and the same newlines are in all other functions). But I don't mind, so I removed it. I will go through these files later and also give comments to all exported functions.